src/Component/Widget/WebComponent.php

```diff
@@ -10,6 +10,18 @@
 
 class WebComponent extends WidgetController
 {
+    public function render()
+    {
+        parent::render();
+        $sForceTemplate = $this->getViewParameter('oxwtemplate');
+
+        if ($sForceTemplate) {
+            $this->_sTemplate = $sForceTemplate;
+        }
+
+        return $this->_sTemplate;
```

Comment:
`_sTemplate` is not defined in any of parent class so it has to be defined in this class
* do not use `_` for property names
* better not use this convention `sForceTemplate` . where the first letter of variable indicates the type. I know its widely used in Oxid code base but not in the module.